### DISCUSSION TITLE
Using ImageMagick to extract clipping path from transparent PNG

### DISCUSSION THREAD

**Discussion by JonasEklundh** [ImageMagick/ImageMagick#3563] [Help]
Using the CSS property clip-path and shape-outside you can make text wrap inside an image container. Examples of the end result can be [seen here](https://medium.com/@nicke920/wrapping-content-around-images-using-css-shapes-2fb17c8fde4e). Example code:

```
.wine — left {
 float: left;
 clip-path: polygon(0% 0%, 70% 0%, 70% 25%, 100% 55%, 100% 100%, 0% 100%);
 shape-outside: polygon(0% 0%, 70% 0%, 70% 25%, 100% 55%, 100% 100%, 0% 100%);
}
```
Now, is there any way to extract this path from a transparent PNG using Magick?

**Comment by snibgo** [comment]
I'm not sure what you are asking. The final PNG may or may not contain metadata of the two paths.
